### PR TITLE
chore(website): build and deploy theme demo [PART-2]

### PR DESCRIPTION
**Pull request by samouss**
**Summary**

This PR migrate the theme demo on CRA to be able to build and host it with the website. I've also create a script to build the website that:

- build each demo and move the build to `website/dist` (only one at the moment) 
- copy the assets to to `website/dist` (only the logo at the moment)

**Result**

- Storybook: [Link](https://deploy-preview-2009--react-instantsearch.netlify.com/storybook/)
- Demo: [Link](https://deploy-preview-2009--react-instantsearch.netlify.com/examples/default-theme/)

### CODE REVIEW

File: scripts/build-website.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const path = require('path');
+const glob = require('glob');
+const { execSync } = require('child_process');
+
+const websitePath = path.join(__dirname, '..', 'website');
+const websiteDistPath = path.join(websitePath, 'dist');
+const examplesSourcePath = path.join(websitePath, 'examples');
+const examplesDistPath = path.join(websiteDistPath, 'examples');
+
+// const examples = glob.sync(path.join(sourcePath, '*'));
```
Comment (Haroenv) [inline scripts/build-website.js]:
```suggestion
// const examples = glob.sync(path.join(examplesSourcePath, '*'));
```

**Comment by Haroenv** [comment]
Netlify says it works, but the deployment isn't actually found though 🤔

**Comment by samouss** [comment]
> Netlify says it works, but the deployment isn't actually found though 🤔 

Yes it's expected because nothing is present at `/`.

File: scripts/build-website.js (hunk 1 above)
Comment (samouss) [reply scripts/build-website.js]: Good catch, but it will be updated later in any cases.

File: website/examples/default-theme/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# tmp
```
Comment (francoischalifour) [inline website/examples/default-theme/README.md]: This should be renamed.

File: website/examples/default-theme/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "tmp",
```
Comment (francoischalifour) [inline website/examples/default-theme/package.json]: This should be renamed.

File: website/examples/default-theme/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "tmp",
```
Comment (francoischalifour) [inline website/examples/default-theme/public/manifest.json]: This should be renamed.

File: website/examples/default-theme/public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "tmp Sample",
```
Comment (francoischalifour) [inline website/examples/default-theme/public/manifest.json]: This should be renamed.

File: website/examples/default-theme/src/App.js [hunk 6]
```diff
@@ -45,9 +44,11 @@ const Header = () => (
       className="is-logo"
     >
       <img
+        alt=""
```
Comment (francoischalifour) [inline website/examples/default-theme/src/App.js]:
```suggestion
        alt="InstantSearch"
```

**Pull request merged**